src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -55,17 +55,15 @@
                     Value="True" />
             <Setter Property="Grid.IsSharedSizeScope"
                     Value="true" />
-            <Setter Property="HasDropShadow"
-                    Value="True" />
```

Comment:
Do we need to delete this property because of the difference between contextMenu and popup?